traversal/planner/PlannerVertex.java

```diff
@@ -260,7 +260,7 @@ void updateObjective(GraphManager graph) {
             if (!props().labels().isEmpty()) {
                 setObjectiveCoefficient(props().labels().size());
             } else if (props().isAbstract()) {
-                setObjectiveCoefficient(graph.schema().stats().abstractTypeCount());
+                setObjectiveCoefficient(graph.schema().stats().abstractThingTypeCount());
```

Comment:
What's the reason we want to restrict the cost to just `abstract` `thingtypes` and not `types`, here?

Reply:
This was just a changing of names. the implementation remains the same.